spec/models/tag_spec.rb

```diff
@@ -30,12 +30,16 @@
 
   it "turns markdown into HTML before saving" do
     tag.rules_markdown = "Hello [Google](https://google.com)"
-    tag.save
+    perform_enqueued_jobs do
```

Comment:
Same as the above specs: evaluating markdown is done synchronously, so there is no need to `perform_enqueued_jobs`.